app/org/sagebionetworks/bridge/dynamodb/DynamoStudy.java

```diff
@@ -68,6 +68,7 @@
     private List<UploadFieldDefinition> uploadMetadataFieldDefinitions;
     private UploadValidationStrictness uploadValidationStrictness;
     private String consentNotificationEmail;
+    private Boolean consentNotificationEmailVerified;
```

Comment:
We do not have a supportEmailVerified flag. Should we? I don't think we record this, we just check it and store it in Redis. If we want the same, let's file a JIRA.